src/components/AppNavigation/CalendarList/CalendarListItemSharingPublishItem.vue

```diff
@@ -213,21 +216,30 @@ export default {
 			this.showEMailInput = true
 			this.showEMailSending = false
 		},
-		sendLinkViaEMail(event) {
+		async sendLinkViaEMail(event) {
 			this.showEMailLabel = false
 			this.showEMailInput = false
 			this.showEMailSending = true
 
-			// const emailAddress = event.target.querySelector('input[type=text]').value
-			//
-			// // HttpClient.post
-			// event.stopPropagation()
-			//
-			// this.sendingEMailLink = true
-			// this.showEMailLinkInput = false
-			//
-			// const value = event.target.querySelector('input[type=text]').value
-			// console.debug(value)
+			const emailAddress = event.target.querySelector('input[type=text]').value
```

Comment:
I don't see right away why we need to use this ?

Reply:
Because ActionInput emits the actual event, not the value of the input: 
https://nextcloud-vue-components.netlify.com/#/Components/Actions?id=actioninput

Reply:
Kinda dirty. :/